os/binfmt/Kconfig

```diff
@@ -24,6 +24,7 @@ config PATH_INITIAL
 config BINFMT_LOADABLE
 	bool
 	default n
+	depends on APP_BINARY_SEPARATION
```

Comment:
`CONFIG_APP_BINARY_SEPARATION` selects the `CONFIG_BINFMT_LOADABLE`.
It looks dangling.
Could you check this?